malware-family/plugx/match-known-module.yml

```diff
@@ -0,0 +1,63 @@
+rule:
+  meta:
+    name: match known module
+    namespace: malware-family/plugx
+    author: still@teamt5.org
+    scope: function
+    references:
+      - https://circl.lu/assets/files/tr-12/tr-12-circl-plugx-analysis-v1.pdf
+      - https://www.fireeye.com/blog/threat-research/2014/07/pacific-ring-of-fire-plugx-kaba.html
+      - https://www.avira.com/en/blog/new-wave-of-plugx-targets-hong-kong
+    examples:
+      - 64E9F62840DB2F65FC717CFAF99081F9:0x10024950
```

Comment:
can you add this sample to capa-testfiles, please?

Reply:
Created PR @ https://github.com/mandiant/capa-testfiles/pull/122